salt/swap/init.sls

```diff
@@ -0,0 +1,12 @@
+coreutils:
+  pkg:
+    - installed
+
+unmount swaps:
+  cmd.run:
+    - name: /sbin/swapoff -a
+
+comment out swap from fstab:
```

Comment:
I think we have been using `this-kind-of-salt-ids` instead of `this kind of salt ids`.